icons/snowman.svg

```diff
@@ -0,0 +1,21 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M2 9h2V7" />
+  <path d="M7 12 4 9" />
+  <path d="m18 11 2-2" />
+  <path d="M20 7v2h2" />
+  <circle cx="12" cy="6" r="4" />
+  <path d="M12 14h.01" />
+  <path d="M12 18h.01" />
+  <path d="M9 8.7a7 7 0 1 0 6 0" />
+  <path d="M5 22h14" />
```

Comment:
![icon (92)](https://github.com/lucide-icons/lucide/assets/25524993/4e049f14-3c1e-484d-a7bc-67292148e8a6)

```suggestion
  <circle cx="12" cy="6" r="4" />
  <path d="M2 9h2V7" />
  <path d="M6 11.5 4 9" />
  <path d="M18 11.5 20 9" />
  <path d="M20 7v2h2" />
  <path d="M 12 14h0.01" />
  <path d="M 12 18h0.01" />
  <path d="M9 8.7a7 7 0 1 0 6 0" />
  <path d="M5 22h14" />
```

Reply:
With eye and nose: ![icon (93)](https://github.com/lucide-icons/lucide/assets/25524993/a5e53b4c-9a76-42b2-8e2c-b00d2e6c2768)

Reply:
I very much preferred the original design, this one looks more like a chubby little penguin than a snowman.